### PR TITLE
css-brochure-form - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**
css-brochure-form - 1

### CODE REVIEW

File: css-brochure-form/css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h2>Learn More</h2>
+
+    <form>
+      <div>
+        <p>Company Email</p>
```
Comment (shawnkost) [inline css-brochure-form/css-brochure-form/index.html]: This should be a `label`

File: css-brochure-form/css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <input type="email" name="email" placeholder="sally@company.co" />
+      </div>
+      <div>
+        <p>Name</p>
```
Comment (shawnkost) [inline css-brochure-form/css-brochure-form/index.html]: This should be a `label`

File: css-brochure-form/css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <input type="text" name="fullName" placeholder="Sally Smith" />
+      </div>
+      <div>
+        <p>Where are you hiring?</p>
```
Comment (shawnkost) [inline css-brochure-form/css-brochure-form/index.html]: This should be a `label`

File: css-brochure-form/css-brochure-form/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        <input type="text" name="location" placeholder="City, Country" />
+      </div>
+      <div>
+        <p>How many engineers are you hiring in the next 6 months?</p>
+        <label for="numEngineers"></label>
```
Comment (shawnkost) [inline css-brochure-form/css-brochure-form/index.html]: The text in your `p` should be inside the `label`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Roland!

**Comment by restropiajr** [comment]
css-brochure-form - 1

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**